scripts/microgenerator/config.yaml

```diff
@@ -0,0 +1,74 @@
+# config.yaml
+
+# The name of the service, used for variable names and comments.
+service_name: "bigquery"
+
+# A list of paths to the source code files to be parsed.
+# Globs are supported.
+source_files:
+  services:
+    - "google/cloud/bigquery_v2/services/dataset_service/client.py"
```

Comment:
If we're already filtering based on class/method should we just make this a glob for all of bigquery_v2/services now?